### DISCUSSION TITLE
Shizuku在这个软件中有什么作用？

### DISCUSSION THREAD

**Discussion by fxzxmicah** [gkd-kit/gkd#594] [Q&A] [answered]
所需要的Shizuku授权我是否可以直接通过adb授权？我的手机无法使用Shizuku。

**Comment by lisonge** [comment, marked as answer]
![image](https://github.com/gkd-kit/gkd/assets/38517192/ca9c0e91-57b0-4bbd-bc7c-78aaa7f3a594)
 
Android 系统 无障碍服务 有时候无法区分自己处于应用的哪个界面，这个时候需要 shizuku 调用高权限的 api 来获取 界面ID

此时 GKD 只能将这个应用的所有界面的规则都去执行，也就是把其他所有页面的规则放到这个页面执行

此外某些应用会屏蔽无障碍点击事件，这时候需要 shizuku 调用高权限的 api 来模拟点击

如 大麦 的开屏广告，如果你没有 shizuku 授权，是无法自动跳过的

---

> 所需要的Shizuku授权我是否可以直接通过adb授权？

不可以

**Reply by fxzxmicah** [reply]
那我再多问一句，就是轻启动申请的是设备管理器权限，这个有帮助吗？毕竟shizuku我是确实用不了。

**Reply by lisonge** [reply]
你好，据我所知，这个是不能的

**Reply by zhoupyxy** [reply]
你好，shizuku授权后需要一直保持后台吗？

**Reply by lisonge** [reply]
shizuku 不需要后台运行
